[2023-04-25T08:37:14.006+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: slack_integration.slack_integration_success manual__2023-04-25T08:37:11.922943+00:00 [queued]>
[2023-04-25T08:37:14.012+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: slack_integration.slack_integration_success manual__2023-04-25T08:37:11.922943+00:00 [queued]>
[2023-04-25T08:37:14.012+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-25T08:37:14.012+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-25T08:37:14.013+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-25T08:37:14.021+0000] {taskinstance.py:1309} INFO - Executing <Task(SlackWebhookOperator): slack_integration_success> on 2023-04-25 08:37:11.922943+00:00
[2023-04-25T08:37:14.024+0000] {standard_task_runner.py:55} INFO - Started process 2027 to run task
[2023-04-25T08:37:14.027+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'slack_integration', 'slack_integration_success', 'manual__2023-04-25T08:37:11.922943+00:00', '--job-id', '115', '--raw', '--subdir', 'DAGS_FOLDER/Task4.py', '--cfg-path', '/tmp/tmpdp395xj6']
[2023-04-25T08:37:14.028+0000] {standard_task_runner.py:83} INFO - Job 115: Subtask slack_integration_success
[2023-04-25T08:37:14.076+0000] {task_command.py:389} INFO - Running <TaskInstance: slack_integration.slack_integration_success manual__2023-04-25T08:37:11.922943+00:00 [running]> on host d51842477fac
[2023-04-25T08:37:14.123+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=slack_integration
AIRFLOW_CTX_TASK_ID=slack_integration_success
AIRFLOW_CTX_EXECUTION_DATE=2023-04-25T08:37:11.922943+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-25T08:37:11.922943+00:00
[2023-04-25T08:37:14.125+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/providers/slack/operators/slack_webhook.py:173: UserWarning: Found unexpected keyword-argument(s) 'link_names' in `send` method. This argument(s) have no effect.
  link_names=self.link_names,

[2023-04-25T08:37:14.132+0000] {base.py:73} INFO - Using connection ID 'slack' for task execution.
[2023-04-25T08:37:14.779+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=slack_integration, task_id=slack_integration_success, execution_date=20230425T083711, start_date=20230425T083714, end_date=20230425T083714
[2023-04-25T08:37:14.821+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-25T08:37:14.839+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
